src/jobBundle/Controller/JobController.php

```diff
@@ -174,37 +194,79 @@ public function getJobsForUpdate()
      */
     public function updateJob($job_id,Request $request)
     {
-         $jobs = new jobEntity();  
-         $em = $this->getDoctrine()->getManager();
+        if (!$this->jobService->Updatejob($job_id)) {
+            
+           return new Response('There is no such an ID '.$job_id );
+           
+        } else {
+            
          $jobs = $this->getDoctrine()
```

Comment:
You should not call doctrine from a controller when you use a service for the same.